src/nupic/algorithms/Cells4.hpp

```diff
@@ -406,8 +407,8 @@ namespace nupic {
                Real permInc =.1,
                Real globalDecay =0,
                bool doPooling =false,
-               int seed =-1,
-               bool doItAll =false,
+               int seed =42,
```

Comment:
@scottpurdy Although, the former case is also a mismatch, with `nupic/TP` 
https://github.com/numenta/nupic/blob/master/nupic/research/TP10X2.py#L90 
(I still have open issue for SDR mismatches, probably caused by unmatching defaults) 
...but it's not a core of this PR, so I will revert if you think it's better (?)